src/components/geocoder-panel.js

```diff
@@ -122,13 +132,8 @@ export default function GeocoderPanelFactory() {
         bbox
       } = geoItem;
       this.removeGeocoderDataset();
-      this.props.updateVisData(
-        [generateGeocoderDataset(lat, lon, text)],
-        {
-          keepExistingConfig: true
-        },
-        PARSED_CONFIG
-      );
+      const updateVisDataPayload = getUpdateVisDataPayload(lat, lon, text);
```

Comment:
you don't need to create a redundant variable. You can do this
```
this.props.updateVisData(...getUpdateVisDataPayload(lat, lon, text));
```

Reply:
At first, I did that but then I wanted to be more readable from my perspective so I added it to a variable. Will fix that.